packages/vda-web3-client/test/didregistry.test.ts

```diff
@@ -402,4 +402,47 @@ describe('DidRegistry Test', () => {
       );
     });
   });
+
+  describe('Manual gas configuration test', () => {
```

Comment:
This is testing the gasConfig being specified as a param.

Can we have tests for `methodDefaults` and `web3config` settings?

Reply:
Global gas configuration & `methodDefaults` is already defiend in the "test/utils.ts" file at line#86 ~ line#95

Reply:
They are defined there, but you need to tests that show the `web3config` is used if the other options aren't provided and that `methodDefaults` is used if gas config isn't specified in `params`.

Reply:
Already tested for various cases. Please check `didregistry.test.ts` file again:
line#411 : `No gas configuration` is test for default web3 config. 
line#425 : `Call wigh global gas configuration` : Used the global gas cofiguration
line#459 : `Call with method default gas configuration` : Used the methodDefaults configuration with global configuration defined
line#513 : `Call with manual gas configuration` : Used to test gas configuration in the function parameters

Reply:
Got it! I missed that. Looks great :)